src/components/Navigation/Navigation.js

```diff
@@ -52,6 +52,13 @@ const List = styled.ul`
   list-style: none;
   display: flex;
 
+.sc-fznKkj fQkkzS:hover{
```

Comment:
Those are not permanent classes. We're using `styled-component` which generate the classes on the fly (so this code isn't working on my environment)

Reply:
ok i will give them specific className may it works